packages/patternfly-4/react-core/src/experimental/components/Spinner/Spinner.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Spinner'
+cssPrefix: 'pf-c-spinner'
+typescript: true
+propComponents: ['Spinner']
+section: 'experimental'
+---
```

Comment:
For this to show up properly on -org, we are using a new `stage: 'early'` in the frontmatter. I'll address this in a follow-up PR.